project/Http4sPlugin.scala

```diff
@@ -26,20 +26,10 @@ object Http4sPlugin extends AutoPlugin {
     },
     scalaVersion := (sys.env.get("TRAVIS_SCALA_VERSION") orElse sys.env.get("SCALA_VERSION") getOrElse "2.12.3"),
 
-    scalacOptions in Compile ++= Seq(
-      "-Yno-adapted-args", // Curiously missing from RigPlugin
-      "-Ypartial-unification", // Needed on 2.11 for Either, good idea in general
-      "-Ywarn-unused-import"
-    ) ++ {
-      // https://issues.scala-lang.org/browse/SI-8340
-      CrossVersion.partialVersion(scalaVersion.value) match {
-        case Some((2, n)) if n >= 11 => Seq("-Ywarn-numeric-widen")
-        case _ => Seq.empty
-      }
-    },
-    scalacOptions in (Compile, console) ~= { xs: Seq[String] =>
-      xs.filterNot(Set("-Xfatal-warnings", "-Ywarn-unused-import"))
-    },
+    // Rig will take care of this on production builds.  We haven't fully
+    // implemented that machinery yet, so we're going to live without this
+    // one for now.
+    scalacOptions := "-Xcheckinit"
```

Comment:
The changes to `HttpDate` were identified by `-Xcheckinit`, which sbt-tpolecat enables by default.  This is great for test builds, but adds runtime overhead to production.  I turned it back off, but we should have this enabled exactly where scoverage is enabled.

Verizon/sbt-rig#19 will do the right thing when we publish from Travis, which is the right thing to do.  I think we should fix this when we start publishing from Travis.